### ISSUE TITLE
TypeError: Cannot read property 'typeAliasInfo' of undefined

### ISSUE THREAD

**Issue by christopherhesse** [microsoft/pyright#1271]
Note: if you are reporting a wrong signature of a function or a class in the standard library, then the typeshed tracker is better suited for this report: https://github.com/python/typeshed/issues.

**Describe the bug**

pyright crashes on a particular file

**To Reproduce**

```py
# copied from pytorch module.pyi
from typing import Callable, TypeVar, Generic


T = TypeVar('T')
T_co = TypeVar('T_co', covariant=True)


class Module(Generic[T_co]):
    __call__: Callable[..., T_co]


def f(function: Callable) -> None:
    pass

f(Module())
```

`pyright`

```
No configuration file found.
stubPath /Users/csh/Downloads/test/typings is not a valid directory.
Assuming Python platform Darwin
Searching for source files
Found 1 source file
An internal error occurred while type checking file "/Users/csh/Downloads/test/pyright_fail.py": TypeError: Cannot read property 'typeAliasInfo' of undefined
    at C (/usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:470811)
    at C (/usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:471246)
    at Object.t.buildTypeVarMapFromType (/usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:481755)
    at wn (/usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:437286)
    at Pn (/usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:435298)
    at Je (/usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:322669)
    at /usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:322781
    at Object.c [as mapSubtypes] (/usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:467197)
    at Ze (/usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:322763)
    at Qe (/usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:322540)
Error performing analysis: TypeError: Cannot read property 'typeAliasInfo' of undefined
    at C (/usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:470811)
    at C (/usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:471246)
    at Object.t.buildTypeVarMapFromType (/usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:481755)
    at wn (/usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:437286)
    at Pn (/usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:435298)
    at Je (/usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:322669)
    at /usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:322781
    at Object.c [as mapSubtypes] (/usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:467197)
    at Ze (/usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:322763)
    at Qe (/usr/local/lib/node_modules/pyright/dist/pyright-internal.js:1:322540)
```

**Expected behavior**

Even if the annotations are wrong (I'm not certain they are right), I would not expect this to crash.

**Screenshots or Code**

Provided above

**VS Code extension or command-line**

command line

```
pyright --version
pyright 1.1.94
```

**Additional context**

N/A

**Comment by erictraut** [comment]
Thanks for the bug report. This bug is one that I happened to already find and fix since the last pyright release. It will be fixed in the next release.

**Comment by jakebailey** [comment]
This is unrelated, but it's curious to me what the source mappings I set up have not kicked in for this installation. I've never seen pyright installed to a system directory; is this being packaged somewhere that's forgetting to copy them?

**Comment by erictraut** [comment]
This is now addressed in pyright 1.1.95, which I just published. It was already included in the last release of pylance.
